README.md

```diff
@@ -109,6 +123,24 @@ Hovering over any resource, class declaration or other Puppet symbol provides in
 
 ![hover_support](https://raw.githubusercontent.com/lingua-pupuli/puppet-vscode/master/docs/assets/hover_support.gif)
 
+### Outline View
+
+The new `Outline View` shows the symbol tree of the currently selected Puppet manifest. This currently supports classes, types, variables and parameters. More symbols are planned to be added.
```

Comment:
Pseudo nit: One sentence per line (semantic linefeeds). This makes review (and editing different) much easier.

The last sentence should be reworked - maybe something like a check box list with the possible symbols and only those which are implemented are checked? Or a link to the issue? Or drop the sentence entirely and just update the section when new symbols are added?,

Reply:
While I generally agree on linefeeds, the whole document does not follow that proscription. I'm disinclined to acquiesce to your request here, and instead volunteer you to copy-edit the readme :)

Will work on last sentence

Reply:
```suggestion
The new `Outline View` shows the symbol tree of the currently selected Puppet manifest.
This currently supports classes, types, variables and parameters.
```